spec/rails_helper.rb

```diff
@@ -71,6 +72,7 @@
   end
 
   config.before(:suite) do
+    Rails.application.load_tasks
```

Comment:
It would work, but then the change by itself isn't necessary. I can update the commit note with why this line had to be added to make things a bit clearer.